projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -173,6 +173,13 @@ public static function load_wpcom_user_features() {
 	 * Can be removed once the feature no longer exists in the ETK plugin.
 	 */
 	public static function load_etk_features_flags() {
+		// Don't load on agency sites.
+		if ( is_fully_managed_agency_site() ) {
```

Comment:
Would this prevent ETK features from loading on the frontpage of A4A sites?
Looking at the comment added below, seems that we want to load them on the frontend page.

Reply:
Hmmmmmmm.... you're right. `is_fully_managed_agency_site()` automatically entails `! is_wpcom_user()`, so for admin pages (like `page-new.php`), this change should have no effect 🤔 

But I'm not sure why it worked when I tested this change via Beta Tester (even before the last commit). 😅

Reply:
Should we revert this particular change?

Reply:
Yes. Do we want to load any features on the frontend page of A4A sites?

Reply:
> But I'm not sure why it worked when I tested this change via Beta Tester (even before the last commit). 😅

I was also curious about this 😆

Reply:
> Do we want to load any features on the frontend page of A4A sites?

Hmmm probably not. Can't think of any use case for it.